### DISCUSSION TITLE
File naming and network description question

### DISCUSSION THREAD

**Discussion by medlyze** [CMSgov/price-transparency-guide#550] [Q&A]
How should we interpret the below string that appears in In-network file names and description fields?

For example:
“266|38F0”

Does “266” represent Provider Group ID?
Does “38F0” represent service code?

In some cases plans use the string within the file name, and in other cases plans use the string as a description of the file.

**Comment by thsummer22** [comment]
It seems that the first three digits ("266") are the plan numbers which map to specific plan names (see this link [https://www.health.ny.gov/statistics/sparcs/sysdoc/appl.htm]).
I am also unsure of what the second half of the string ("38F0") represents.

**Reply by medlyze** [reply]
Thank you @thsummer22 for providing the reference link to the BCBSA plan code. We are still confirming the remaining digits which may be a plan-specific code.

**Reply by thsummer22** [reply]
@Pksegp99 Were you ever able to confirm if the remaining digits are plan-specific codes?

**Reply by medlyze** [reply]
Sorry for delay @thsummer22. We are not yet certain but should have confirmation shortly!
